v8/import_page/import_page.py

```diff
@@ -61,34 +60,52 @@ class CommandImportPage(Command):
 
     def _execute(self, options, args):
         """Import a Page."""
-        if libextract is None:
-            utils.req_missing(['libextract'], 'use the import_page plugin')
+        if BeautifulSoup is None:
+            utils.req_missing(['bs4'], 'use the import_page plugin')
         for url in args:
             self._import_page(url)
 
     def _import_page(self, url):
-        r = requests.get(url)
-        if 199 < r.status_code < 300:  # Got it
-            # Use the page's title
-            doc = lxml.html.fromstring(r.content)
-            title = doc.find('*//title').text
-            if sys.version_info[0] == 2 and isinstance(title, str):
-                title = title.decode('utf-8')
+        parse = requests.utils.urlparse(url)
+        if 'http' in parse.scheme:
+            r = requests.get(url)
+            if not (199 < r.status_code < 300):  # Did not get it
+                LOGGER.error(f'Error fetching URL: {url}')
+                return 1
+            html = r.content.decode(r.encoding).encode('utf-8') if r.encoding and 'utf-8' \
+                not in r.encoding.lower() else r.content
```

Comment:
This seems suspicious, why does `html` need to be encoded to `utf-8`?

Reply:
Thank you; it was a remnant from my previous attempt to figure out why a page without 'body' (having 'ISO-8859-1' encoding) did produce empty list for 'nodes'.